reconcile/utils/jobcontroller/controller.py

```diff
@@ -0,0 +1,222 @@
+import logging
+import time
+from typing import TextIO, Tuple, TypeVar
+
+from kubernetes.client import ApiClient, V1Job  # type: ignore[attr-defined]
```

Comment:
this is related to the introduction of the `kubernetes-stubs` dep in `requirements-type.txt` - but let me check again

Reply:
yep, because of the stubs

```
reconcile/utils/oc.py:26: error: Module "kubernetes.client" does not explicitly export attribute "ApiClient"  [attr-defined]
reconcile/utils/oc.py:26: error: Module "kubernetes.client" does not explicitly export attribute "Configuration"  [attr-defined]
```